lib/huginn_acumen_product_agent/acumen_product_agent.rb

```diff
@@ -77,10 +82,10 @@ def handle(event)
 
             # map attributes
             products.map do |product|
-                map_attributes(product)
+                map_contributor_types(product)
```

Comment:
@mrogers-5s did you intentionally remove the attribute mapping?

Reply:
@jspizziri Definitely an accident. Fixed